hurumap_land/templates/homepage.html

```diff
@@ -23,35 +23,78 @@ <h4>{{ HURUMAP.name }} provides useful facts and data about land ownership in So
   </div>
 </div>
 {% endblock header_content %}
+{% block content_container %}
 
-{% block how_it_works %}
-<div class="wrapper clearfix how-it-works">
-  <h1>How It Works</h1>
-  <br/>
-  <div class="column-third">
-    <p><span class="img-bg rounded bg-color-primary"><img src="{% static 'img/icons/blurbs/checklist.svg' %}"/></span></p>
-    <h3>Find Facts</h3>
-    <p>Land ownership data is broken down by province and category: agricultural land, erven and urban land and sectional title land. </p>
+  <div class="clearfix">
+      <div id="slippy-map"></div>
   </div>
-  <div class="column-third">
-    <p><span class="img-bg rounded bg-color-primary"><img src="{% static 'img/icons/blurbs/analysis.svg' %}"/></span></p>
-    <h3>Visualize</h3>
-    <p>Our library of charts gives you insight into land data from provinces you research. Look for them on profile pages. You can embed the charts on your own site. </p>
+  <div class="home-nav text-center section-container bg-color-white">
+    <ul>
+      <li><a>
+          <span class="img-bg rounded text-white" style="padding: 20px 27px;">
+          <img src="{% static 'img/icons/blurbs/papyrus.svg'%}"/>
+          <br/>ABOUT</span></a></li>
```

Comment:
This is missing a link to about page or how it works?

Reply:
@DavidLemayian I have directed the link to about page